Project/core/src/main/java/Controller/GameController.java

```diff
@@ -85,25 +74,18 @@ public GameController(int playerAmount) {
         }
     }
 
-    private Game initializeGame(int playerAmount) {
+    private Game initializeGame(int playerAmount, Csv boardCsv) {
+        if (playerAmount <= 0) {
+            throw new IllegalArgumentException("Must be more than 2 players");
+        }
```

Comment:
Checks for above zero players but error says its checking for two. It is also possible to play with only two people.